<?xml version="1.0" encoding="utf-8"?>
<launch>
    <!-- 加载深度摄像头 -->
    <include file="$(find astra_camera)/launch/astra.launch"/>

    <!-- 加载视觉主控 -->
    <node pkg="wtr_race" type="video_tackle.py" name="video_tackle" output="screen">
        <rosparam file="$(find wtr_race)/config/ost.yaml" command="load" />
    </node>

    <node pkg="wtr_race" type="base2camera.py" name="base2camera" output="screen"/>
    <!-- 加载rviz配置 -->
    <!-- <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find wtr_race)/config/traj.rviz" /> -->

</launch>